Source/Dafny/DafnyOptions.cs

```diff
@@ -1124,6 +1129,14 @@ not sound
         /verificationLogger:trx;LogFileName=<...>.
     The exact mapping of verification concepts to the TRX and CSV formats is
     experimental and subject to change!
+/printAllVerificationResults
+    Prints a summary of all verification results after verification has
+    completed. For every assertion batch of every method, prints the
+    list of assertions that occur within that batch, the status of the
+    batch, and the resources (in terms of time and solver resources)
+    needed analyze the batch. For fine-grained analysis of the result
+    and cost of verifying every assertion, combine this with the Boogie
```

Comment:
Nit (and probably moot given my other suggestions), but we should be careful about implying that we're measuring the true cost of each assertion that accurately. Probably want to sneak "approximate" into any description like this, or go into more detail about how batching (and hence un-batching) can affect the cost unpredictably.